### ISSUE TITLE
Accept zero as valid for transforms

### ISSUE THREAD

**Issue by arackaf** [pmndrs/react-spring#1049]
## 🚀 Feature Proposal

This is super minor, and I'll completely understand if you just close it as "won't implement" but I bumped into it so I figured it was at least worth bringing up.

```js
  const transition = useTransition(booksObj, {
    config: { ...config.default }, //config.molasses,
    from: { opacity: 0, position: "static", transform: "translate3d(0, 0px, 0px)" },
    enter: { opacity: 1, position: "static", transform: "translate3d(0, 0px, 0px)" },
    leave: { opacity: 0, position: "absolute", transform: "translate3d(25%, 0px, 0px)" }
  });
```

The transform above does not work, since the zero's in the X slot have no units, even though zero would normally be considered a dimensionless quantity.

The workaround is trivial of course

```js
  const transition = useTransition(booksObj, {
    config: { ...config.default }, //config.molasses,
    from: { opacity: 0, position: "static", transform: "translate3d(0%, 0px, 0px)" },
    enter: { opacity: 1, position: "static", transform: "translate3d(0%, 0px, 0px)" },
    leave: { opacity: 0, position: "absolute", transform: "translate3d(25%, 0px, 0px)" }
  });
```

But it took me a minute or two to figure this out

## Motivation

See above

## Example

See above

**Comment by aleclarson** [comment]
I think a lot of people trip on this. 

The problem is, the current "string interpolation" logic is generic, so I'm a little uncertain how that could be fixed, but I guess adding another global would be good enough. Then the `@react-spring/web` package could inject its own handling for transform strings.

Btw, you probably don't want to write out the transform string manually, since you can use shorthand style props instead: https://aleclarson.github.io/react-spring/v9/#Shorthand-style-props

**Comment by arackaf** [comment]
Whoa - nice tip on the shorthand form - thanks a ton!

**Comment by joshuaellis** [comment]
released in v9.3.0
